scripts/set_vars.sh

```diff
@@ -12,7 +12,19 @@ export REPO_PATH="${REPO_PATH:-${PLATFORM_PATH}/repo.path}"
 export SC_MACHINE_REPO="${SC_MACHINE_REPO:-https://github.com/ostis-ai/sc-machine.git}"
 export SC_MACHINE_BRANCH="${SC_MACHINE_BRANCH:-main}"
 export SC_MACHINE_PATH="${SC_MACHINE_PATH:-${PLATFORM_PATH}/sc-machine}"
+export SC_MACHINE_NAME="${SC_MACHINE_NAME:-sc-machine}"
 
 export SC_WEB_REPO="${SC_WEB_REPO:-https://github.com/ostis-ai/sc-web.git}"
 export SC_WEB_BRANCH="${SC_WEB_BRANCH:-main}"
 export SC_WEB_PATH="${SC_WEB_PATH:-${PLATFORM_PATH}/sc-web}"
+export SC_WEB_NAME="${SC_WEB_NAME:-sc-web}"
+
+if [ -d "${SC_MACHINE_PATH}" ];
+then
+  source "${SC_MACHINE_PATH}/scripts/set_vars.sh"
```

Comment:
`SC_WEB_NAME` is used to clone the repo, but `SC_WEB_PATH` should be used to indicate the clone path.

Reply:
Fixed